script.js

```diff
@@ -12,4 +14,182 @@ function closePopUp(){
+
+  },
+
+  {
+    prName: 'SignIn and signUp Demo',
+    prDesc: 'The site gives you knowhow ,how to put sign and sign section as a popup in a website',
+    techList: ['HTML', 'CSS', 'Javascript'],
+    backgroundImg: './images/sm-pr-login1.jfif',
+    pg:'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    buttonLive: 'See Live',
+    liveLink: 'https://codewithhyder.github.io/logindemo/',
+    buttonSource: 'See Source',
+    sourceLink: 'https://github.com/codeWithHyder/logindemo'
+
+  },
+
+  {
+    prName: 'Birth Day Reminder',
+    prDesc: 'Keeping track of hundreds of components website',
+    techList: ['HTML', 'CSS', 'React'],
+    backgroundImg: './images/sm-pr-birthday-reminder.png',
+    pg:'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    buttonLive: 'See Live',
+    liveLink: 'https://codewithhyder.github.io/bdayreminder1/',
+    buttonSource: 'See Source',
+    sourceLink: 'https://github.com/codeWithHyder/bdayreminder1'
+
+  },
+
+];
+
+//link see project buttons to project info array respectively
+
+cardButtonsArray.forEach((buttonItem, index)=>{
+  const items=projectInfo[index];
+  //start popup window
+  buttonItem.addEventListener('click',()=>{
+    
+  //blur the website background
+  headerSection.classList.toggle('blur');
+  heroSection.classList.toggle('blur');
+  workSection.classList.toggle('blur');
+  aboutMeSection.classList.toggle('blur');
+  contactFormSection.classList.toggle('blur');
+  footerSection.classList.toggle('blur');
+
+  //add markup for modal
+  const bodyEle=document.querySelector('body');
+  const section=document.createElement('section');
+  bodyEle.appendChild(section);
+  section.innerHTML=`<div class="modal" id="modal">
+  
+  <div class="modal-header" id="modal-header">
+    <h2>${items.prName}</h2>
+    
+    <button type=""button" class="cross-button" id="cross-button">&times;</button>
+  </div>
+  <div class="tech-list" id="tech-list">
+    <span class="tech-list1">${items.techList[0]}</span>
+    <span class="tech-list1">${items.techList[1]}</span>
+    <span class="tech-list1">${items.techList[2]}</span>
+  </div>
+  <div class="modal-body" id="modal-body">
+  
+      <div class="modalbody-left"><img src="${items.backgroundImg}" alt='featureimg'></div>
+      <div class="modalbody-right">
+        <h3>${items.pg}</h3> 
+         
+          <div class="modal-source-live">
+            <a href="${items.liveLink}"><button type="button" class="livebtn">See Live <img src="./images/live-icon.png"></button></a>
+            <a href="${items.sourceLink}"><button type="button" class="sourcebtn">See Source <img src="./images/git-icon.png"></button></a>
+          </div>
```

Comment:
- [ ] Kindly reduce the font sizes for the buttons so that it matches the provided Figma design.

**This is how your design looks like:**

![image](https://user-images.githubusercontent.com/86318284/217455982-62aaaeee-47b6-418b-8a4e-b7c7af2c2906.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/217455920-3e80469c-ab6c-4ab4-ab1a-d2c5ecf47034.png)